lib/QueryRunner.js

```diff
@@ -45,18 +45,25 @@ export default class QueryRunner {
       })
     }
 
+    let portOffsetArray = []
+    if (Array.isArray(gameQueryPortOffset)) {
+      portOffsetArray = gameQueryPortOffset
+    } else if (gameQueryPortOffset) {
+      portOffsetArray.push(gameQueryPortOffset)
```

Comment:
Even simpler would be:
```js
let portOffsetArray = gameQueryPortOffset
if (!Array.isArray(portOffsetArray)) { // the given offset is not an array apparently, it's a single value, so lets fix that
  portOffsetArray = [gameQueryPortOffset]
}
```
Sorry if I seem to be a little petty about this, just thought it could be better (:

Reply:
Please, by all means! Your input is very much appreciated!

Reply:
I did some testing and when we don't have `port_query_offset` present, we get `gameQueryPortOffset` set as `undefined` and this will mess with the attempts for when we don't have a port set by the user.

```js
} else if (gameOptions.port) {
      portOffsetArray.forEach((portOffset) => { addAttemptWithPort(gameOptions.port + portOffset) })
}
```

The line `portOffsetArray.push(0)` was basically setting a `[0]` to the `gameQueryPortOffset`.
So if I remove `port_query_offset` from `games.js` and run this below, I'll get an error.

`node bin/gamedig --type vrising 45.137.247.66`

Reply:
I've updated to the code below, so we make sure to always have a `0` to be used when no port is provided.
Let me know if it makes sense! (: 

```js
let portOffsetArray = gameQueryPortOffset
if (!Array.isArray(portOffsetArray)) {
  gameQueryPortOffset ? portOffsetArray = [gameQueryPortOffset] : portOffsetArray = [0]
}
```

Reply:
great catch, thanks.